### PR TITLE
Sync with runtime flags from standalone

### PR DESCRIPTION
**Pull request by VourMa**
As per title. It goes together with SegmentLinking/TrackLooper#408.

### CODE REVIEW

File: RecoTracker/LST/plugins/alpaka/LSTProducer.cc [hunk 1]
```diff
@@ -77,14 +81,16 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE {
       desc.add<edm::InputTag>("pixelSeedInput", edm::InputTag{"lstPixelSeedInputProducer"});
       desc.add<edm::InputTag>("phase2OTHitsInput", edm::InputTag{"lstPhase2OTHitsInputProducer"});
       desc.add<int>("verbose", 0);
+      desc.add<int>("nopLSDupClean", 0);
+      desc.add<int>("tcpLSTriplets", 0);
```
Comment (slava77) [inline RecoTracker/LST/plugins/alpaka/LSTProducer.cc:85]: why `int` ?

Comment (VourMa) [reply RecoTracker/LST/plugins/alpaka/LSTProducer.cc:85]: Consistency with `verbose_` (which is not well justified). But I can take the opportunity and move everything to `bool` in this PR.

Comment (slava77) [reply RecoTracker/LST/plugins/alpaka/LSTProducer.cc:85]: verbosity is frequently multi-leveled (from quiet to `very-*-very` verbose). The yes/no options are not.

Comment (slava77) [reply RecoTracker/LST/plugins/alpaka/LSTProducer.cc:85]: OTOH, `verbose` is a `bool` in the `LST::run`. So, makes sense to keep them all `bool`

Comment (VourMa) [reply RecoTracker/LST/plugins/alpaka/LSTProducer.cc]: If we decide to go with a more elaborate verbosity scheme, let's switch back to the appropriate type. For now, I switched everything to `bool`, as proposed.

**Review by slava77 — APPROVED** [review]

**Comment by VourMa** [comment]
As discussed in the meeting, I will use this PR at some point next week as the testing ground for the workflow of development within CMSSW.

**Comment by VourMa** [comment]
Superseded by #37.

**Pull request closed without merging**